src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1445,6 +1445,12 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
 
         $container->getDefinition('messenger.sender_locator')->replaceArgument(0, $senderLocatorMapping);
         $container->getDefinition('messenger.asynchronous.routing.sender_locator')->replaceArgument(1, $messageToSenderIdsMapping);
+
+        if ($config['doctrine']['enabled']) {
+            $container->getDefinition('messenger.middleware.doctrine_transaction')->replaceArgument(1, $config['doctrine']['entity_manager']);
```

Comment:
`$config['doctrine']` should be `$config['doctrine_transaction']` and `['entity_manager']` should be `['entity_manager']['name']`, we will need tests for this if it's not already covered

Reply:
Thank you